sources/_base_sources/other/near_base_sources.yml

```diff
@@ -227,6 +227,67 @@ sources:
             description: Ingestion timestamp
             type: timestamp
 
+      - name: ft_transfer_calls
```

Comment:
i don't think we need to add this to source file, since your only source is actions?
this is a model, and already defined in schema yml file